src/huggingface_hub/repository.py

```diff
@@ -655,6 +656,12 @@ def clone_from(self, repo_url: str, use_auth_token: Union[bool, str, None] = Non
                                 " on Hugging Face Hub."
                             )
                         else:
+                            warnings.warn(
+                                "Creating a repository through clone_from is deprecated"
+                                "will be removed in v0.10.",
```

Comment:
Maybe let's change this to v0.11., as 0.9 is already the next release, that way we have 2 releases before deprecating